src/ApiService/ApiService/Functions/Containers.cs

```diff
@@ -16,11 +16,12 @@ public class ContainersFunction {
 
     [Function("Containers")]
     [Authorize(Allow.User)]
-    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "GET", "POST", "DELETE")] HttpRequestData req)
+    public Async.Task<HttpResponseData> Run([HttpTrigger(AuthorizationLevel.Anonymous, "GET", "POST", "PUT", "DELETE")] HttpRequestData req)
```

Comment:
PATCH might more appropriate here since the container is partially updated instead of Replaced